src/lib/format/l20n/ast/parser.js

```diff
@@ -7,18 +7,20 @@ const MAX_PLACEABLES = 100;
 
 export default {
   parse: function(env, string, pos = false) {
```

Comment:
yeah, is parseResource ok? (trying to name it similar to parseProgram from JS and parseStylesheet from CSS)